src/kitconcept/volto/behaviors/configure.zcml

```diff
@@ -20,4 +20,11 @@
       provides=".navtitle.INavTitle"
     />
 
+  <plone:behavior
+    name="volto.headtitle"
```

Comment:
@sneridagh I would go with "head_title" here. "Head Title" is the right term in English, so it should have a blank/underscore. This is also consistent with what we used in fzj so far.